src/main/java/blackjack/dto/GamerDto.java

```diff
@@ -1,27 +1,27 @@
 package blackjack.dto;
 
-import java.util.List;
-
 import blackjack.domain.gamer.Dealer;
 import blackjack.domain.gamer.Gamer;
 
+import java.util.List;
+
 public record GamerDto(
-    String name,
-    CardsDto cards
+        String name,
+        CardsDto cards
 ) {
 
     public static GamerDto from(Gamer gamer) {
         return new GamerDto(
-            gamer.getName(),
-            new CardsDto(gamer.getCards().stream()
-                .map(CardDto::from)
-                .toList()));
+                gamer.getName(),
+                new CardsDto(gamer.getCards().stream()
+                        .map(CardDto::from)
+                        .toList()));
     }
 
     public static GamerDto dealerFrom(Dealer dealer) {
         return new GamerDto(
```

Comment:
```suggestion
    public static GamerDto from(Dealer dealer) {
        return new GamerDto(
```

이미 인자로 Dealer임을 알 수 있어서 생략해주는게 나아보입니다.

Reply:
동의합니다, 해당 부분 적용하겠습니다! 다형성을 활용한다는게 어떤건지 조금씩 와닿는 것 같습니다 🤓

Reply:
[refactor(gamer): 메서드명 개선](https://github.com/woowacourse/java-blackjack/pull/908/commits/279733befce61db5eb1f92e444bbf3eb87c195d3)